brianlong/www.validators.app
# 178177826_update_schema_rb

#### What's this PR do?
- Update collation to utf8mb4_0900_ai_ci in schema.rb

#### How should this be manually tested?
- Run migration, check if there is no changes in schema.rb
- Remember to add 
 ``` 
encoding: utf8mb4
collation: utf8mb4_0900_ai_ci
```

to your database.yml (you need to have mysql 8.0 installed or higher version for this collationj)

#### What are the relevant tickets?
- https://www.pivotaltracker.com/story/show/178177826

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
